docs/howto/how_to_launch_Apollo.md

```diff
@@ -4,43 +4,41 @@ nVidia GPU and that you have installed the Linux nVidia drivers.
 You could still proceed with the next few steps even if you are missing the Linux nVidia drivers, as the system will run but with the CUDA-based perception and other modules.
 ```
 # To get a list of build commands
-./apollo.sh
-# To make sure you start clean
-./apollo.sh clean
-# This will build the full system and requires that you have an nVidia GPU with nVidia drivers loaded
-bash apollo.sh build
+bash apollo.sh
+# To make sure you start clean, cleanup bazel output and log/coredump files.
+bash apollo.sh clean
+# This will build the full system and requires that you have an nVidia GPU with nVidia drivers loaded.
+# You can specify a module to build, e.g. <module> = cyber or modules/<module>. If <module> unspecified, build all.
+bash apollo.sh build [module]
+# And you also can specify a module to build with the explict options, eg. dbg, opt, cpu, gpu.
+# Run debug build (default)
```

Comment:
1) `./apollo.sh build` doesn't default to `./apollo.sh build --config=dbg` anymore.
2) Split the code section into multiple ones.